airbyte-config/specs/build.gradle

```diff
@@ -32,4 +33,18 @@ task generateConnectorSpecsMask(type: JavaExec, dependsOn: generateSeedConnector
 
 project(":airbyte-config:init").tasks.processResources.dependsOn(generateConnectorSpecsMask)
 
+task generateCombinedConnectorCatalog(type: JavaExec, dependsOn: generateSeedConnectorSpecs) {
```

Comment:
New gradle task - you can either call `gw :airbyte-config:init:generateCombinedConnectorCatalog` or the usual `gw :airbyte-config:init:processResources` since it's added as a dependency